pkg/diff/diff_test.go

```diff
@@ -56,16 +56,16 @@ func toUnstructured(obj interface{}) (*unstructured.Unstructured, error) {
 
 func mustToUnstructured(obj interface{}) *unstructured.Unstructured {
 	un, err := toUnstructured(obj)
-	errors.CheckError(err)
+	errors.CheckErrorWithCode(err, errors.ErrorCommandSpecific)
 	return un
 }
 
 func unmarshalFile(path string) *unstructured.Unstructured {
 	data, err := ioutil.ReadFile(path)
-	errors.CheckError(err)
+	errors.CheckErrorWithCode(err, errors.ErrorCommandSpecific)
 	var un unstructured.Unstructured
 	err = json.Unmarshal(data, &un.Object)
-	errors.CheckError(err)
+	errors.CheckErrorWithCode(err, errors.ErrorCommandSpecific)
```

Comment:
Same here as with above - tests should just use `CheckError()` (or even better, I think `panic()` would be the right thing to do in tests, but we should think about this -- CC @alexmt)